docs/en/repository.md

```diff
@@ -60,6 +60,7 @@ Those operators are:
 - `<=`
 - `>`
 - `>=`
+- `like`
```

Comment:
add the like example to the other operation examples below ;-)

Reply:
And now? :)